src/server/services/import/import.js

```diff
@@ -81,6 +110,54 @@ const importOtherSlug = async (data, { slug, user, idField }) => {
   };
 };
 
+const assertRequiredFields = (data, requiredFields) => {
+  const keys = Object.keys(data);
+  if (requiredFields.some((key) => !keys.includes(key))) {
+    throw new Error('Missing required field');
```

Comment:
For easier tracking of errors, could you add the name of the missing field to the error message and figure out a way to also show which data row is the offending one. Maybe just add to add `data` to the output as well or catch this error in the outermost loop and add row number to the message?